src/main/java/pw/chew/chanserv/listeners/MessageModificationHandler.java

```diff
@@ -54,26 +69,36 @@ public void onMessageUpdate(@NotNull MessageUpdateEvent event) {
         // Ignore webhooks
         if (event.getMessage().isWebhookMessage()) return;
 
-        // grab message from map
+        // Retrieve the old message from the cache
         FanclubMessage message = messagesMap.get(event.getMessageId());
 
-        // Put new message into map
+        // Cache the updated message
         messagesMap.put(event.getMessageId(), new FanclubMessage(event.getMessage()));
 
-        if (message == null) {
-            return;
-        }
+        // Skip logging if the message wasn't cached
+        if (message == null) return; 
 
+        // Store messages
         String oldMessage = message.content();
         String newMessage = event.getMessage().getContentRaw();
 
+        // Fetch the original creation date
+        OffsetdateTime creationTime = TimeUtil.getTimeCreated(message.getId());
+        String creationDateTime = creationTime.format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss"));
```

Comment:
We should use `TimeFormat` here. Keep creationTime and just do `TimeFormat.DATE_TIME_SHORT.format(creationTime)`

Example: https://github.com/Chewbotcca/Chewbotcca-Discord/blob/main/src/main/java/pw/chew/chewbotcca/commands/info/ServerInfoCommand.java#L119
Docs: https://docs.jda.wiki/net/dv8tion/jda/api/utils/TimeFormat.html